ported/gatsby-starter/assets/flashload.js

```diff
@@ -0,0 +1,394 @@
+window.FlashLoad = (function() {
```

Comment:
You do not want to add `flashload.js`. I will remove it from Ghost-attila too. Flashload is a default asset.

`/assets/flashload.js` works for all blogs. :)

Reply:
done